pkg/agent/nodeportlocal/rules/iptable_rule.go

```diff
@@ -106,35 +107,29 @@ func (ipt *iptablesRules) AddAllRules(nplList []PodNodePort) error {
 	return nil
 }
 
-// DeleteRule deletes a specific NPL rule from NodePortLocalChain chain
-func (ipt *iptablesRules) DeleteRule(port int, podIP string) error {
+// DeleteRules deletes a specific NPL rule from NodePortLocalChain chain
+func (ipt *iptablesRules) DeleteRules(port int, podIP string) error {
```

Comment:
Was this renaming (`DeleteRule` -> `DeleteRules`) intentional? it doesn't seem so

Reply:
Thank you for catching this. It was changed mistakenly when I changed another `DeleteRules` methods.